api/src/main/java/org/openmrs/ConceptNumeric.java

```diff
@@ -45,6 +46,8 @@ public class ConceptNumeric extends Concept {
 	
 	private Boolean allowDecimal = false;
 	
+	private List<ConceptReferenceRange> conceptReferenceRanges;
```

Comment:
Since this is already in the concept numeric class, we can shorten this to `referenceRanges` and avoid repeating `concept`.